datastream-server/src/main/java/com/linkedin/datastream/server/assignment/LoadBasedTaskCountEstimator.java

```diff
@@ -27,6 +54,30 @@
    */
   public int getTaskCount(ClusterThroughputInfo throughputInfo, List<String> assignedPartitions,
       List<String> unassignedPartitions) {
-    throw new NotImplementedException();
+    Validate.notNull(throughputInfo, "null throughputInfo");
+    Validate.notNull(assignedPartitions, "null assignedPartitions");
+    Validate.notNull(unassignedPartitions, "null unassignedPartitions");
+    LOG.info("Assigned partitions: {}", assignedPartitions);
```

Comment:
We need throughput info as well. There is no easy way for us to get this info otherwise.

Reply:
@vmaheshw let me think about this. I don't want to dump this in the log file. It can be huge.